### ISSUE TITLE
Support QOI image format

### ISSUE THREAD

**Issue by duarm** [mapeditor/tiled#3849]
**Is your feature request related to a problem? Please describe.**
Tiled doesn't support the QOI image format

**Describe the solution you'd like**
Tiled supports QOI

**Describe alternatives you've considered**
I'm currently keeping both a PNG for tiled, and a QOI for my engine, I keep both in sync

**Comment by duarm** [comment]
Today I tried importing a QOI file and it worked no problem. Not sure if it was an update on Tiled's side, or installing qt6-imageformats.

**Comment by bjorn** [comment]
@duarm Right, or you have [kimageformats](https://github.com/KDE/kimageformats) installed, which seems to add the necessary Qt image format plugin for this. These plugins might also be part of the Flatpak runtime Tiled uses, in case you're using the Flatpak version.

In any case it's great to see QOI getting adoption and while it might be supported now in your case, I don't think support for this exists for the macOS and Windows binaries I've released so far. Worth looking into.
